src/Exception/JiraTicketNotFoundException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace duckpony\Exception;
+
+class JiraTicketNotFoundException extends \Exception
```

Comment:
```suggestion
class JiraTicketNotFoundException extends \RuntimeException
```

Please always use a more specific exception as base for custom ones.